app/uk/gov/hmrc/cgtpropertydisposals/service/returns/CgtCalculationService.scala

```diff
@@ -38,7 +38,8 @@ trait CgtCalculationService {
     reliefDetails: CompleteReliefDetailsAnswers,
     exemptionAndLosses: CompleteExemptionAndLossesAnswers,
     estimatedIncome: AmountInPence,
-    personalAllowance: AmountInPence
+    personalAllowance: AmountInPence,
+    initialGainOrLossDetails: Option[AmountInPence]
```

Comment:
I don't think it's going to break anything, but it's slightly annoying that this is called `initialGainOrLossDetails` here in the trait and then `initialGainOrLoss` later on in the class

Reply:
Changed to initialGainOrLoss